pkg/backup/constants/constants.go

```diff
@@ -55,4 +55,7 @@ const (
 
 	// ServiceAccountCAPath is where is CABundle of serviceaccount locates
 	ServiceAccountCAPath = "/var/run/secrets/kubernetes.io/serviceaccount/ca.crt"
+
+	// KMS secret env prefix
+	KMSSecretPrefix = "KMS_ENCRYPTED"
```

Comment:
how this is being used? any docs or links about this env?

Reply:
the dbaas encrpytion secret with KMS so the  secret mount to pod should decryption with [shush](https://github.com/realestate-com-au/shush) this prefix means this env should decryption